invoices/invoiceregistry_test.go

```diff
@@ -1326,10 +1332,10 @@ func TestMultipleSetHeightExpiry(t *testing.T) {
 	ctx := newTestContext(t, nil)
 
 	// Add a hold invoice.
-	invoice := *testInvoice
-	invoice.HodlInvoice = true
+	testInvoice := newInvoice(t, false)
```

Comment:
Q: Why don't we just do `newInvoice(t,true)` but want a hodl Invoice with a preimage set?